### PR TITLE
Add Debug to token traits `AssetId` and `Balance`

### PR DESCRIPTION
**Pull request by KingFishDev**
Fixes [#167](https://github.com/paritytech/subport/issues/167)

### CODE REVIEW

File: frame/support/src/traits/tokens/misc.rs [hunk 1]
```diff
@@ -160,9 +161,9 @@ impl WithdrawReasons {
...
-pub trait AssetId: FullCodec + Copy + Default + Eq + PartialEq {}
-impl<T: FullCodec + Copy + Default + Eq + PartialEq> AssetId for T {}
+pub trait AssetId: FullCodec + Copy + Default + Eq + PartialEq + Debug {}
+impl<T: FullCodec + Copy + Default + Eq + PartialEq + Debug> AssetId for T {}
 
 /// Simple amalgamation trait to collect together properties for a Balance under one roof.
-pub trait Balance: AtLeast32BitUnsigned + FullCodec + Copy + Default {}
-impl<T: AtLeast32BitUnsigned + FullCodec + Copy + Default> Balance for T {}
+pub trait Balance: AtLeast32BitUnsigned + FullCodec + Copy + Default + Debug {}
+impl<T: AtLeast32BitUnsigned + FullCodec + Copy + Default + Debug> Balance for T {}
```
Comment (shawntabrizi) [inline frame/support/src/traits/tokens/misc.rs:169]: I think `RuntimeDebug` is better :P

**Review by shawntabrizi — APPROVED** [review]

File: frame/support/src/traits/tokens/misc.rs (hunk 1 above)
Comment (xlc) [reply frame/support/src/traits/tokens/misc.rs:169]: RuntimeDebug is a derive macro, not a trait.

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
@KingFishDev please sign the cla.

File: frame/support/src/traits/tokens/misc.rs (hunk 1 above)
Comment (shawntabrizi) [reply frame/support/src/traits/tokens/misc.rs:169]: ah my bad :P

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**